### PR TITLE
AB#9051 Full page search

### PR DESCRIPTION
**Pull request by sc-nelikostadinova**

### CODE REVIEW

File: src/rendering/src/pages/_app.tsx [hunk 1]
```diff
@@ -55,6 +55,24 @@ function App({ Component, pageProps, router }: AppPropsWithLayout): JSX.Element
   });
   // END CUSTOMIZATION
 
+  // DEMO TEAM CUSTOMIZATION - Search SDK integration
+  useEffect(() => {
+    if (isSearchSDKEnabled) {
+      PageController.getContext().setLocaleLanguage('en');
+      PageController.getContext().setLocaleCountry('us');
+      trackEntityPageViewEvent('content', {
```
Comment (adoprog) [inline src/rendering/src/pages/_app.tsx:63]: Don't forget to pull latest updates from 'develop' - path.tsx already has tracking.

https://github.com/Sitecore/Sitecore.Demo.XmCloud.PlaySummit/blob/develop/src/rendering/src/pages/%5B%5B...path%5D%5D.tsx

File: src/rendering/src/components/ContentSearch/SearchFacets.tsx [hunk 2]
```diff
@@ -36,94 +28,81 @@ const getFacetLabel = ({ text }: FacetValue): string => {
...
-                  ))}
-                </AccordionFacets.ValueList>
-              </AccordionFacets.Content>
-            </AccordionFacets.Facet>
-          </div>
+              </AccordionFacets.Trigger>
+            </AccordionFacets.Header>
+            <AccordionFacets.Content className="search-facets-root-facet-content">
+              <AccordionFacets.ValueList className="search-facets-root-facet-list">
+                {f.value?.map((v, index) => (
```
Comment (chpa7) [inline src/rendering/src/components/ContentSearch/SearchFacets.tsx:85]: Maybe sort the facet values alphabetically?

File: src/rendering/src/components/ContentSearch/SearchFacets.tsx [hunk 3]
```diff
@@ -36,94 +28,81 @@ const getFacetLabel = ({ text }: FacetValue): string => {
...
+                    {...{ index, facetValueId: v.id }}
+                    className="search-facets-root-facet-item"
+                  >
+                    <AccordionFacets.ItemCheckbox className="search-facets-root-facet-checkbox">
+                      <AccordionFacets.ItemCheckboxIndicator className="search-facets-root-facet-checkbox-indicator">
+                        <FontAwesomeIcon icon={faCheck} />
+                      </AccordionFacets.ItemCheckboxIndicator>
+                    </AccordionFacets.ItemCheckbox>
+                    <AccordionFacets.ItemLabel className="search-facets-root-facet-checkbox-label">
+                      {getFacetLabel(v)} {v.count && `(${v.count})`}
```
Comment (chpa7) [inline src/rendering/src/components/ContentSearch/SearchFacets.tsx:97]: And capitalize them? Would look a bit more consistent.

**Review by chpa7 — APPROVED** [review]
Looks great!

**Pull request merged**